### DISCUSSION TITLE
ERROR in ./node_modules/@tanstack/query-core/build/modern/mutation.js

### DISCUSSION THREAD

**Discussion by shubhamanandspeqto** [wevm/wagmi#4113] [Connector Request]
### What **novel use-case** does the Connector provide?

Hey folks! I'm have install @tanstack/react-query in our typescript code for integrating walletconnect but after installing updated @tanstack/react-query it through error i.e ERROR in ./node_modules/@tanstack/query-core/build/modern/mutation.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
SyntaxError: /home/adminshubham/Downloads/BDEX/Interface-2/node_modules/@tanstack/query-core/build/modern/mutation.js: Class private methods are not enabled. Please add @babel/plugin-proposal-private-methods to your configuration.
![Screenshot from 2024-07-04 13-27-55](https://github.com/wevm/wagmi/assets/91736262/00e215e6-2650-4ea2-9192-1022beef17cd)

### Are the Connector's integrations production-ready and generally available?

![Screenshot from 2024-07-04 13-30-27](https://github.com/wevm/wagmi/assets/91736262/097bb469-e92b-4d4a-8a6e-0303eaf897d3)

### Are you committed to actively maintaining the Connector?

- [X] Yes, my team is or I am committed to actively maintaining the Connector.

**Comment by shawnlauzon** [comment]
I hit the same issue using react native. Did you ever find a solution to this?

**Reply by mateusdegobi** [reply]
worked for me:

```
rm -rf node_modules
rm -rf package-lock.json yarn.lock
yarn install
npx react-native start --reset-cache
```
